src/disco/css/Addon.scss

```diff
@@ -196,6 +196,10 @@ $addon-padding: 20px;
     right: 0;
     top: 0;
 
+    [dir=rtl] & {
+      padding: 15px 20px 15px 80px;
```

Comment:
I think it would be great to add some helpers for RTL stuff. This is pretty straightforward but it is also duplicating these styles.

I made some JSBins with what seems useful for this:

[Before mixins](https://jsbin.com/zeyudos/1/edit?css,output)
[With mixins](https://jsbin.com/zeyudos/3/edit?css,output)

I'm not sure how powerful our CSS linter is but if we start using these mixins we could probably add a rule to throw when a `padding` or `margin` is set with different left and right values or when `left` or `right` is used.

Mixins:

```scss
@mixin bidi-padding($top, $end, $bottom, $start) {
  padding: $top $end $bottom $start;
  
  [dir=rtl] & {
    padding: $top $start $bottom $end;
  }
}

@mixin bidi-start($offset) {
  left: $offset;
  right: auto;
  
  [dir=rtl] & {
    left: auto;
    right: $offset;
  }
}

@mixin bidi-end($offset) {
  left: auto;
  right: $offset;
  
  [dir=rtl] & {
    left: $offset;
    right: auto;
  }
}